Source/Core/Core/HW/SI_DeviceGCController.cpp

```diff
@@ -26,6 +26,27 @@ CSIDevice_GCController::CSIDevice_GCController(SIDevices device, int _iDeviceNum
 {
 	// Dunno if we need to do this, game/lib should set it?
 	m_Mode                   = 0x03;
+
+	m_Calibrated = false;
+}
+
+void CSIDevice_GCController::GetOrigin()
```

Comment:
This may want to be renamed something different, since the `Get` part implies it's returning something. Maybe `CalibrateOrigin()` or `UpdateOrigin()`?